Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -126,18 +172,191 @@ void GameList::OpenProperties()
   properties->show();
 }
 
+void GameList::ExportWiiSaves()
+{
+  QMessageBox result_dialog(this);
+  if (GameFile(GetSelectedGame()).ExportWiiSaves())
+  {
+    result_dialog.setIcon(QMessageBox::Information);
+    result_dialog.setText(tr("Successfully exported save files"));
+  }
+  else
+  {
+    result_dialog.setIcon(QMessageBox::Critical);
+    result_dialog.setText(tr("Failed to export save files!"));
+  }
+  result_dialog.exec();
+}
+
 void GameList::OpenWiki()
 {
   QString game_id = GameFile(GetSelectedGame()).GetGameID();
   QString url = QStringLiteral("https://wiki.dolphin-emu.org/index.php?title=").append(game_id);
   QDesktopServices::openUrl(QUrl(url));
 }
 
+void GameList::CompressISO()
+{
+  const auto original_path = GetSelectedGame();
+  auto file = GameFile(original_path);
+
+  const bool compressed = (file.GetBlobType() == DiscIO::BlobType::GCZ);
+
+  QString label_text = tr("Select where you want to save the ");
```

Comment:
Building strings in this way is unacceptable when it comes to localization. (We don't localize DolphinQt2 right now, but I'm assuming we want it to be able to handle it in the future since you're using `tr` here.) You should at least use something like %s instead, but it would probably be even better if you just had two complete versions of the string.